docs/user/getting-startedv2.rst

```diff
@@ -0,0 +1,116 @@
+
+Note that Maestro2 also features a web API presented in :ref:`web-api`.
+
+Setup
+-----
+The first step is to install the dependencies along with the tools and resources that are to be used
+
+- Download and install Java 11
+- Download the :download:`example resources part1 <images/wt_example/watertank-example-part1.zip>` and unzip into a directory.
+- Download the :download:`example resources part2 <images/wt_example/watertank-example-part2.zip>` and unzip into a directory.
+- Download `maestro-2.x.x-jar-with-dependencies.jar` from the newest `Maestro2 2.x.x` release on the Maestro release page https://github.com/INTO-CPS-Association/maestro/releases and place it in the same directory as the example resources were unzipped into.
+
+.. _getting_started_part1:
+
+Part 1: First MaBL Specification
+------------------------
+The example below concerns a water tank. The tank (Continuous Time component) has a constant inflow of water and a valve that is controlled by a controller (Discrete-Event component). When the valve is open, the water level within the tank decreases, and whe the valve is closed, the water level increases.
+The controller determines when to open and close the valve based on a maximum and a minimum water level.
+
+.. _fig-watertankexample:
+.. figure:: images/wt_example/wt-example.png
+    :align: center
+
+    Water Tank Example
+
+The corresponding MaBL code, including descriptive comments, to execute this co-simulation is presented below:
+
+.. literalinclude:: images/wt_example/wt-example.mabl
+    :language: c
+
+To execute this (one can use the file corresponding wt-example.mabl from the unzipped example resouces) run the following command from the terminal:
+
+.. code-block:: none
+
+    > java -jar maestro-2.x.x-jar-with-dependencies.jar --interpret wt-example.mabl
+        where --interpret is to interpret a specification
+
+The result is available in `outputs.csv`. This can be plotted with `pythoncsvplotter.py`, and the result should look like the figure below.
+
+.. _fig-watertankexample-result:
+.. figure:: images/wt_example/wt-example-result.png
+    :align: center
+
+    Result from co-simulation
+
+Part 2: Specification Generation and Expansion
+----------------------------------------------
+This part also concerns the water tank but uses the capabilities of specification generation and expansion to create the specification. Thus, the user does not have to write MaBL by hand. Specification generation and Expansion are not treated in detail in this guide, but more information is available in :ref:`Specification Generation` and :ref:`Expansion`.
+
+The specification generator is based on JSON configuration files with the same structure as the ones used in Maestro1. For this reason, it is also possible to use the `INTO-CPS Application <https://into-cps-association.readthedocs.io/projects/desktop-application/en/latest/>` to create the specification files.
```

Comment:
Maybe it might be nice to know which of the files thats present in the R_ folder that can be used. i.e. if the initialise and simulation files are given as input with -sg1 then it will execute it